### PR TITLE
cen64: init at unstable-2020-02-20

### PR DESCRIPTION
**Pull request by 414owen**
###### Motivation for this change

cen64 is an emulator that's useful for n64 development

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113750` at 8d117ad7 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/de80c5406f06ffdaa3857132f71dc898">cen64</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/misc/emulators/cen64/default.nix:17:3:
```
   |
17 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -82,6 +82,12 @@
     githubId = 882455;
     name = "Elliot Cameron";
   };
+  _414owen = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:85]: Please put this into a separate commit with the message ``maintainers: add _414owen``.

File: pkgs/misc/emulators/cen64/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{ lib, cmake, fetchFromGitHub, libGL, libiconv, libX11, openal, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "cen64";
+  version = "20200220-git";
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/cen64/default.nix:5]:
```suggestion
  version = "unstable-2020-02-20";
```

File: pkgs/misc/emulators/cen64/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  installPhase = ''
+    mkdir -p $out/bin
+    mv cen64 $out/bin
+  '';
+
+  meta = with lib; {
+    description = "A Cycle-Accurate Nintendo 64 Emulator";
+    license = licenses.bsd3;
+    homepage = "https://www.cen64.com/";
+    maintainers = [ lib.maintainers._414owen ];
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/cen64/default.nix:26]:
```suggestion
    maintainers = [ maintainers._414owen ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
